javacpp-android-gradle/app/ndk-stl-config.cmake

```diff
@@ -0,0 +1,39 @@
+# Copy shared STL files to Android Studio output directory so they can be
```

Comment:
Did you copy this from somewhere? If so, please add the URL of the original here in comments! Thanks

Reply:
@saudet sure!

The source is from the google's oboe repository:

https://github.com/google/oboe/blob/master/samples/hello-oboe/src/main/cpp/ndk-stl-config.cmake